src/EIPTestsFiller/StateTests/stEOF/stEIP4200/CreateTransactionInvalidEOF1Filler.yml

```diff
@@ -0,0 +1,184 @@
+      # - Jump to another RJUMPI immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(5) STOP PUSH1(1) RJUMPI(-11) STOP - 0xef0001010004020001000c030000000000000160015d00050060015dfff500
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010004020001000c030000000000000160015d00050060015dfff500" }'
+      # - Jump to RJUMP Immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(3) STOP RJUMP(-9) - 0xef00010100040200010009030000000000000160015d0003005cfff7
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef00010100040200010009030000000000000160015d0003005cfff7" }'
+      # - Jump to PUSH Immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(-4) STOP - 0xef00010100040200010006030000000000000160015dfffc00
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef00010100040200010006030000000000000160015dfffc00" }'
+      # - Jump to RJUMPV immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(5) STOP PUSH1(1) RJUMPV(0) STOP - 0xef0001010004020001000d030000000000000160015d00050060015e01000000
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010004020001000d030000000000000160015d00050060015e01000000" }'
+      # - Jump to CALLF immediate
+      # Code to be deployed: 0xef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1
+      #   Code[0]: PUSH1(1) RJUMPI(2) CALLF(1) STOP
+      #   Code[1]: PUSH1(1) PUSH1(1) SSTORE RETF
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1" }'
+      # Legacy Initcode trying to deploy invalid EOF code containing RJUMPV with count 0, and zero immediates
+      # Code to be deployed: PUSH1(1) RJUMPI( - 0xef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1" }'
+      # Legacy Initcode trying to deploy invalid EOF code containing RJUMPV with count 0, and one immediate
+      # Code to be deployed: PUSH1(1) RJUMPI( - 0xef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1" }'
+      # Legacy Initcode trying to deploy invalid EOF code containing truncated RJUMPV
+      # Code to be deployed: PUSH1(1) RJUMPI( - 0xef00010100040200010004030000000000000160015e01
```

Comment:
```suggestion
      # Code to be deployed: PUSH1(1) RJUMPV( - 0xef00010100040200010004030000000000000160015e01
```

Reply:
Makes sense to add also the case where the last byte is `5e`, i.e. `RJUMPV` with no immediates